client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -41,8 +43,14 @@ const Email = React.createClass( {
 	},
 
 	render() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		const classes = classNames( 'domain-management-email', {
+			'is-wide-layout': showPlanFeatures
```

Comment:
Sorry had a misread on the `has-personal-plans` class. As we chatted on slack, this  is being used on the existing plans page. Lets avoid a refactor in this case since it will be removed soon.